ndx-multichannel-volume/ndx-meta.yaml

```diff
@@ -0,0 +1,8 @@
+name: ndx-multichannel-volume
+version: 0.1.0
+src: https://github.com/focolab/ndx-multichannel-volume
+pip: https://pypi.org/project/ndx-multichannel-volume/
```

Comment:
Please update the `README.md` in the extension repo so that users who encounter the repo source or the PyPI page know what the extension is and does.